shopyo/api/cli.py

```diff
@@ -446,32 +446,29 @@ def new(projname, verbose):
     click.echo(f"[x] Project {projname} created successfully!\n")
 
 
-@cli.command("rundebug", with_appcontext=False)
-def rundebug():
-    """runs the shopyo flask app in development mode"""
+def run_helper(mode=""):
```

Comment:
Any reason why you pass `mode` as key value argument rather than just do "`run_helper(mode)`"?

Reply:
hi, thought that might be better since mode can be of any type other than string for input value. Is there a case where it is beneficial to pass mode as key value rather than just mode?

Reply:
Actually the `mode` can only be of type string (because [`create_app`](https://github.com/shopyo/shopyo/blob/dev/shopyo/app.py#L30) argument `config_name` should only accept string as well). One of the things we need to do in future is to add typing which makes code less error prone but if we were to add typing to this function then it would be as follows
```python
def run_helper(mode: str) -> None:
    pass
```